toolkit/offchain/src/scripts_data.rs

```diff
@@ -34,7 +34,6 @@ pub struct PolicyIds {
 	pub permissioned_candidates: PolicyId,
 	pub reserve_auth: PolicyId,
 	pub version_oracle: PolicyId,
-	pub governance: PolicyId,
```

Comment:
This wasn't used apparently, because the value that we've returned was invalid - see what was removed in line 85 and comment in tests in line 249